src/api/volunteerRequest.ts

```diff
@@ -13,7 +15,7 @@ const router = Router();
 /**
  * @openapi
  * paths:
- *   /api/v1/volunteer-requests:
+ *   /api/v1/volunteer-requests/volunteer:
```

Comment:
`/api/v1/volunteer-requests/volunteer` collides with `/api/v1/volunteer-requests/{requestId}`.
i think this route should stay as it was since its the route that hands all of the volunteer-requests that are relevant for the site